src/View/Helper/FlashMessenger.php

```diff
@@ -167,14 +152,14 @@ protected function renderMessages(
             return '';
         }
 
+        $objNamespace = $this->getDefaultNamespace();
+        if (isset($this->namespaces[$namespace])) {
```

Comment:
If this is an array key existence check, let's use `array_key_exists()`.

Also, we can use `$objNamespace = $this->namespaces[$namespace] ?? $this->getDefaultNamespace();`